sphinx/templates/latex/tabular.tex_t

```diff
@@ -1,3 +1,11 @@
+<%- if table.align -%>
+\begin{table}[<%= table_align %>]
```

Comment:
Note:

For figure directive, `:align:` option makes the figure float.
>align : "left", "center", or "right"
The horizontal alignment of the figure, allowing the image to float and have the text flow around it. The specific behavior depends upon the browser or rendering software used.

But, for table directives, the definition does not mention about float. So we should not wrap text around the table even if aligned.
>align : "left", "center", or "right"
The horizontal alignment of the table. (New in Docutils 0.13)
http://docutils.sourceforge.net/docs/ref/rst/directives.html#table